bazel/cargo/wasmtime/Cargo.toml

```diff
@@ -10,8 +10,8 @@ path = "fake_lib.rs"
 env_logger = "0.9"
```

Comment:
`env_logger` should be updated to `0.10` to match upstream.

(from https://github.com/bytecodealliance/wasmtime/blob/v9.0.2/Cargo.toml)